MWSE/LuaManager.cpp

```diff
@@ -2434,6 +2434,74 @@ namespace mwse::lua {
 		OnSkillTrained_SkillId = -1;
 	}
 
+	void LuaManager::gatherModMetadata() {
+		sol::table luaMWSE = luaState["mwse"];
+		auto activeLuaMods = luaMWSE["activeLuaMods"];
+
+		// Try to match any mwse information from -metadata.toml files to an active lua mod.
+		for (const auto& p : std::filesystem::directory_iterator("Data Files", std::filesystem::directory_options::follow_directory_symlink)) {
+			auto lowerPath = p.path().string();
+			string::to_lower(lowerPath);
+
+			// We only care *-metadata.toml files.
+			if (!string::ends_with(lowerPath, "-metadata.toml")) {
+				continue;
+			}
+
+			// Load the metadata.
+			sol::safe_function_result metadata_result = luaState["toml"]["loadFile"](lowerPath);
+			if (metadata_result.valid()) {
+				sol::table metadata = metadata_result;
+				if (metadata == sol::nil) {
+					continue;
+				}
+
+				sol::table metadata_tools = metadata["tools"];
+				if (metadata_tools == sol::nil) {
+					continue;
+				}
+
+				sol::table metadata_tools_mwse = metadata_tools["mwse"];
+				if (metadata_tools_mwse == sol::nil) {
+					continue;
+				}
+
+				sol::optional<std::string> luaKey = metadata_tools_mwse["lua-mod"];
```

Comment:
I think this needs a `string::to_lower`, otherwise it only works if the `lua-mod` field in the metadata is all lowercase.

Reply:
Fixed in 4360278264962cfb848ee4fea13d646dab2dcf67.